tools/experimental/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 set(MCRL2_TOOLS
   besconvert  
+  complps2pbes
```

Comment:
This is probably not a good idea. There's a reason this experimental tool is not included in the targets. Does it even compile?

Reply:
Okay, this is related to me not having the documentation for pbesabstract. I compiled mCRL2 locally and it did not compile the complps2pbes and pbesabstract tools. Because the documentation build process scans the `src/tools/experimental` directory to determine which tools to generate help pages for, it does not have help for these tools.

My solution for fixing the missing complps2pbes man page was to add it here. Is there a better way to compile these two tools? Some CMake flag I have overlooked?

Reply:
I think it would be best to imitate the behaviour of the old scripts: scan the `tools/{experimental,release}` directories and try to run each tool to obtain a man page. If a tool cannot be run and no RST file exists for it, no tool page is generated. Is this solution feasible?

Reply:
Yes, that's the current approach. I will revert this specific change.

Reply:
I do wonder though - how does the website have tool documentation for these tools currently? They must be compiled somewhere. Or is that cached?

Reply:
Reverted this change in 7fe4bdca279cf79af15851b93ec9fd982855a5ea.

Reply:
> How does the website have tool documentation for these tools currently?
I don't see the pages you are referring to. On https://mcrl2.org/web/user_manual/tools.html, there is no link for complps2pbes and the page for pbesabstract does not contain a man page.